app/utils/alerts.py

```diff
@@ -288,9 +288,13 @@ def build_alerts_elements(images_url_live_alerts, live_alerts, map_style):
     alerts_markers_layer = dl.LayerGroup(children=alerts_markers, id="alerts_markers")
 
     # Building the alerts notification btn
-    nb_alerts = len(all_events)  # Number of unique events
+    today = pd.Timestamp.now().normalize()
+    today_alerts_df = all_events[all_events["created_at"] >= today]
```

Comment:
Both look good but maybe using always the same type of comparison (`>= today` or `day == today`) would be clearer ?